### ISSUE TITLE
Github Action: SSH into Droplet

### ISSUE THREAD

**Issue by XiXiaPdx** [AntonEmery/chord-app#42]
research is #40 

Successfully add github action step to ssh into droplet 
Have it depend on step in #41  , publish to Dockerhub
Need to add the following secrets (these refer to DO values) to github

```
    HOST: ${{ secrets.HOST }}
          USERNAME: ${{ secrets.USERNAME }}
          KEY: ${{ secrets.SSHKEY }}
```

Verify SSH is successful with a `ls` of root directory of Droplet.

**Comment by XiXiaPdx** [comment]
@AntonEmery  We will need those secrets placed into Github as well. Thanks!

**Comment by AntonEmery** [comment]
Closed by #45
